modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -1332,8 +1480,14 @@ export default class I3SConverter {
    */
   private createAttributeStorageInfo(
     tileContent: Tiles3DTileContent | null,
-    propertyTable: FeatureTableJson | null
-  ): void {
+    propertyTable: FeatureTableJson | null,
+    dumpedAttributeTypesMap?: Record<string, Attribute> | null
+  ): Record<string, Attribute> | null {
+    if (dumpedAttributeTypesMap) {
+      // Add new storage attributes, fields and create popupInfo
+      this.attributeMetadataInfo.addMetadataInfo(dumpedAttributeTypesMap);
```

Comment:
The idea was: 
* the dump is updated here; 
* `attributeMetadataInfo` is restored once in the start of the conversion

Reply:
I got the idea, but just forgot to remove changes in this method. Done now.